packages/scripts/scripts/test-e2e.js

```diff
@@ -0,0 +1,44 @@
+// Do this as the first thing so that any code reading it knows the right env.
+process.env.BABEL_ENV = 'test';
+process.env.NODE_ENV = 'test';
+
+// Makes the script crash on unhandled rejections instead of silently
+// ignoring them. In the future, promise rejections that are not handled will
+// terminate the Node.js process with a non-zero exit code.
+process.on( 'unhandledRejection', ( err ) => {
+	throw err;
+} );
+
+/**
+ * External dependencies
+ */
+const jest = require( 'jest' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	fromConfigRoot,
+	getCliArgs,
+	hasCliArg,
+	hasProjectFile,
+	hasJestConfig,
+} = require( '../utils' );
+
+// Provides a default config path for Puppeteer when jest-puppeteer.config.js
+// wasn't found at the root of the project or a custom path wasn't defined
+// using JEST_PUPPETEER_CONFIG environment variable.
+if ( ! hasProjectFile( 'jest-puppeteer.config.js' ) && ! process.env.JEST_PUPPETEER_CONFIG ) {
+	process.env.JEST_PUPPETEER_CONFIG = fromConfigRoot( 'puppeteer.config.js' );
+}
+
+const hasRunInBand = hasCliArg( '--runInBand' ) ||
+	hasCliArg( '-i' );
+
+jest.run(
```

Comment:
Given the similarity between this and `test-unit-js` I'd perhaps pass the arguments to jest in the same way (config and cli arguments variables) as to lessen the comparison effort between the two files.

Reply:
They are different though. For e2e tests you have to provide this `-i` flag which makes all of this complex.

I can align with the other one if that makes it read easier.

Reply:
Updated to align with the other file, let me know what you think.

Reply:
I like it more :)